build-system/babel-plugins/babel-plugin-transform-internal-version/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Reassigns the trailing comments of a statement to be leading comment of its
+// next sibling. This is because JSDoc comments (which should be on the next
+// statement) get erroneously assigned as trailing comments of this statement.
+module.exports = function (babel, options = {}) {
+  const {types: t} = babel;
+  const {version} = options;
+
+  if (!version) {
+    throw new Error(
+      'Need to specify runtime version as an option to babel transformer'
+    );
+  }
+
+  return {
+    name: 'ast-transform', // not required
+    visitor: {
+      StringLiteral(path) {
+        const {node} = path;
+        const {value} = node;
+        if (!value.includes('$internalRuntimeVersion$')) {
```

Comment:
Do we want to check `value === ` or `value.includes`?

Reply:
With `.includes`, we could run this transform on test files and fix cases like this:

https://github.com/ampproject/amphtml/blob/4ebe517fe6dbb2038298bbecae3e715576610be0/extensions/amp-analytics/0.1/test/test-iframe-transport.js#L185

Reply:
That'd be great! Just need to add it to the babel config I think

Reply:
Yup, that was the intention, but I was going to do that in a separate PR in case the tests failed.